pkg/controller/account/account_controller.go

```diff
@@ -925,3 +925,72 @@ func getBuildIAMUserErrorReason(err error) (string, awsv1alpha1.AccountCondition
 		return "UnhandledError", awsv1alpha1.AccountUnhandledError
 	}
 }
+
+// getManagedTags retrieves a list of managed tags from the configmap
+// returns an empty list on any failure.
+func (r *ReconcileAccount) getManagedTags(log logr.Logger) []awsclient.AWSTag {
+	tags := []awsclient.AWSTag{}
+
+	cm := &corev1.ConfigMap{}
+	err := r.Client.Get(context.TODO(), types.NamespacedName{Namespace: awsv1alpha1.AccountCrNamespace, Name: awsv1alpha1.DefaultConfigMap}, cm)
+	if err != nil {
+		log.Info("There was an error getting the default configmap.", "error", err)
+		return tags
+	}
+
+	managedTags, ok := cm.Data[awsv1alpha1.ManagedTagsConfigMapKey]
+	if !ok {
+		log.Info("There are no Managed Tags defined.")
+		return tags
+	}
+
+	return parseTagsFromString(managedTags)
+}
+
+// getCustomerTags retrieves a list of customer-provided tags from the linked accountclaim
+func (r *ReconcileAccount) getCustomTags(log logr.Logger, account *awsv1alpha1.Account) []awsclient.AWSTag {
+	tags := []awsclient.AWSTag{}
+
+	accountClaim, err := r.getAccountClaim(account)
+	if err != nil {
+		log.Error(err, "Error getting AccountClaim to get custom tags")
+		return tags
+	}
+
+	if accountClaim.Spec.CustomTags == "" {
+		return tags
```

Comment:
I think since managed tags are probably going to be defined on every cluster and they're the going to be the same across all instances, I'm going to leave the message the way it is.  Managed Tags not being defined means that every cluster doesn't get tags when they should, where there's only a very small amount of clusters that will define custom tags, so that would spam the logs a lot more and not really add much value.

Granted, if there were no managed tags defined it would spam the logs just as much, but I think that gets offset because the managed tags _should always_ be defined.  

I'm not quite sure if that makes sense the way it's written or not, but I'm happy to clarify further.